docs/concepts/architecture/master-node-communication.md

```diff
@@ -106,3 +108,83 @@ cluster (connecting to the ssh server listening on port 22) and passes all
 traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the private
 GCE network in which the cluster is running.
+
+
+
+## Configuring communication ports
+
+> A note on the difference between specifying 0.0.0.0 or 127.0.0.1 as a bind `--address`. *0.0.0.0* will bind to ALL interfaces on the host. *127.0.0.1* will ONLY bind to localhost.
+
+
+
+
+### kube-apiserver
+
+| Argument                       | Default           | Description                              |
+| ------------------------------ | ----------------- | ---------------------------------------- |
+| --advertise-address            | --bind-address    | The IP address on which to advertise the apiserver to members of the cluster. This address must be reachable by the rest of the cluster |
```

Comment:
the double-hypen options are not very visible in the table format (see https://deploy-preview-7111--kubernetes-io-master-staging.netlify.com/docs/concepts/architecture/master-node-communication/) - I recommend considering wrapping them in a code style (that is `--advertise-address`, `--bind-address`, etc)